sdk/servicebus/service-bus/README.md

```diff
@@ -43,23 +50,26 @@ You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your t
 
 ### Authenticate the client
 
-Interaction with Service Bus starts with an instance of the [ServiceBusClient](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.1/classes/servicebusclient.html) class.
+Interaction with Service Bus starts with an instance of the [ServiceBusClient][sbclient] class.
 
 You can instantiate this class using its constructors:
 
-- [Create using a connection string](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.1/classes/servicebusclient.html#constructor)
+- [Create using a connection string][sbclient_constructor]
   - This method takes the connection string to your Service Bus instance. You can get the connection string
     from the Azure portal.
-- [Create using a TokenCredential](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.1/classes/servicebusclient.html#constructor)
+- [Create using a TokenCredential][sbclient_constructor]
   - This method takes the host name of your Service Bus instance and a credentials object that you need
     to generate using the [@azure/identity](https://www.npmjs.com/package/@azure/identity)
     library. The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
 
 ### Key concepts
 
-Once you have initialized the [ServiceBusClient](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-service-bus/7.0.0-preview.1/classes/servicebusclient.html)
-class, create a sender or receiver based on whether you want to send or receive messages.
-to interact with existing Service Bus entities.
+Once you have initialized the [ServiceBusClient][sbclient] class, you will create a sender or receiver
+based on whether you want to send or receive messages to interact with existing Service Bus entities.
+
+- To send messages, to a queue or topic, you'll use an instance of [`Sender`][sender] which
```

Comment:
Its odd that in the previous sentence we talk about creating sender/receiver and we do that again in the line here.

I would avoid calling out "an instance of Sender" due to the worry that people might try to new it up.

How about a bulleted list for the Key Concepts like we have for keyvault-keys: https://www.npmjs.com/package/@azure/keyvault-keys#key-concepts

Reply:
- ServiceBusClient is the entry point to the package
- Use the sender from the getSender() method on the ServiceBusClient to send messages to queue or topic
- Use the receiver from the getReceiver() method on the ServiceBusClient to receive messages from a queue or subscription

Reply:
@ramya-rao-a  - I can collapse it into a bullet point list but now that you've pointed me to keyvault-keys...

...@czubair - from our review discussion the other day I was under the impression that this section was supposed to link to key classes.

In the link that @ramya-rao-a 's provided it's a bit more like I would have expected - just a nice textual description, probably portable from language to language.

I've heard some conflicting ideas about what that section should be. 

Can you provide some clarity?

Reply:
There has been differences among different services in different languages. I don't believe we have consistency in the same language either.

The guideline for readme: https://github.com/Azure/azure-sdk/blob/f51fa161f21a152dc6cfa1cb89818c9be090f592/docs/policies/README-TEMPLATE.md#key-concepts

For the sake of this PR, I am ok with either except the repetition that we currently have

Reply:
Okay, that seems to say that what I've got there is good from the content standpoint.

I've made it a bulletpoint list.

Reply:
@ramya-rao-a  - I believe I've addressed your feedback sufficiently and I am awaiting a response from you. Please resolve this comment if you think it's good.